global/scripts/MyelinMap_BC.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+set -eu
+
+pipedirguessed=0
+if [[ "${HCPPIPEDIR:-}" == "" ]]
+then
+    pipedirguessed=1
+    #fix this if the script is more than one level below HCPPIPEDIR
+    export HCPPIPEDIR="$(dirname -- "$0")/.."
+fi
+
+source "$HCPPIPEDIR/global/scripts/newopts.shlib" "$@"
+source "$HCPPIPEDIR/global/scripts/debug.shlib" "$@"         # Debugging functions; also sources log.shlib
+
+defaultSigma=$(echo "sqrt(200)" | bc -l)
```

Comment:
This is a strange default sigma, as it is not in the form of a fwhm-conversion, `FWHM / (2 * sqrt(2 l(2)))`.  What did we previously use?

Reply:
That is real, but at this point I forget why it is that.  Probably something I did over a decade ago.

Reply:
Technically it has a fairly obtuse connection to 200 mm^2 on a plane (a square aligned with one quarter of the kernel, reaching 1 sigma at the non-diagonal points, which could be cut on the other diagonal and rearranged to very approximately cover a bit less than half the 1-sigma circle)...

Reply:
Here're several cases in the existing scripts:
- https://github.com/Washington-University/HCPpipelines/blob/master/PostFreeSurfer/PostFreeSurferPipeline.sh#L50
- https://github.com/Washington-University/HCPpipelines/blob/master/MSMAll/scripts/MSMAll.sh#L529